<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runprocessgui</class>
 <widget class="QMainWindow" name="runprocessgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>runprocessgui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Host Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="username_label">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="username_edit"/>
           </item>
           <item row="1" column="0" rowspan="2">
            <widget class="QLabel" name="hostname_label">
             <property name="text">
              <string>Hostname</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QLineEdit" name="hostname_edit"/>
           </item>
           <item row="3" column="0" rowspan="2">
            <widget class="QLabel" name="port_label">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" rowspan="2">
            <widget class="QLineEdit" name="port_edit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="command_label">
             <property name="text">
              <string>Command</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="command_edit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Authentication options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="auth_none">
           <property name="text">
            <string>SSH None method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="auth_pubkey">
           <property name="text">
            <string>Automatic public key authentication</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="auth_password">
           <property name="text">
            <string>SSH Password method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="auth_kbi">
           <property name="text">
            <string>SSH Keyboard Interactive method</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="run_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>200</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Run Command</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Command Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="output_edit">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="plainText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
